### PR TITLE
Add `last-login-ip` capture capability

### PR DESCRIPTION
**Pull request by kartik-v**
| Q             | A
| ------------- | ---
| Is bugfix?    | no
| New feature?  | yes
| Breaks BC?    | no
| Tests pass?   | yes
| Fixed issues  | self (184)

### CODE REVIEW

**Comment by kartik-v** [comment]
@tonydspaniard - check if you can merge this - will be useful to have this for user activity tracking generically (and moderating)... thanks.

**Comment by maxxer** [comment]
can you please fix failing tests?

**Comment by kartik-v** [comment]
@maxxer the error is because there is a new db field added `last_login_ip` and there is a new migration script that needs to be run (included with this PR). 

```php
[yii\base\UnknownPropertyException] Setting unknown property: Da\User\Model\User::last_login_ip 
```

The changes in this PR have been tested on a DEV environment and after running the migration script.

**Comment by maxxer** [comment]
The test does run migrations

https://travis-ci.org/2amigos/yii2-usuario/jobs/378582131#L701

**Comment by kartik-v** [comment]
> The test does run migrations

Am not sure why the tests are then not reading the `last_login_ip` column added to the db table ... 

All seems to work at fine when I checked on my local... Have to check later on this repo's unit tests when I get time -- if you do find something missed do let know.

**Comment by maxxer** [comment]
Can't recall why but you must add the field to the SQL initialization  script `tests/_data/schema.sql`. Check [here](https://github.com/2amigos/yii2-usuario/pull/136/commits/c004a7c4c117f76ccc0f529caffd50a727d09cc8)

File: tests/_data/schema.sql [hunk 1]
```diff
@@ -73,6 +73,7 @@ CREATE TABLE `user` (
   `auth_tf_enabled` tinyint(1) DEFAULT '0',
   `flags` int(11) NOT NULL DEFAULT '0',
   `last_login_at` int(11) DEFAULT NULL,
+  `last_login_ip` string(45) varchar(45) DEFAULT NULL,
```
Comment (maxxer) [inline tests/_data/schema.sql]: there's an extra `string(45)`

**Comment by maxxer** [comment]
I'll investigate over the failing tests  and merge asap! thanks!

**Pull request merged**